### ISSUE TITLE
Bintray & Artifactory search fails for packages with "=" symbol in the values of settings

### ISSUE THREAD

**Issue by danimtb** [conan-io/conan#5642]
Coming from #5581

Seems that Bintray does not support parsing the conaninfo.txt files that have a content similar to this when doing a search:

```
[settings]
    arch=x86_64
    build_type=Release
    compiler=Visual Studio
    compiler.runtime=MD
    compiler.toolset=v141,version=14.14,cuda=10.0,host=x64
    compiler.version=15
    os=Windows
```

In particular, the line ``compiler.toolset=v141,version=14.14,cuda=10.0,host=x64`` makes the pacakge indexer fail as it is unable to parse ``=`` symbols inside the value for ``compiler.toolset`` (the commas ``,`` are parsed ok)

I think this was fixed in earlier versions of the package indexer and that it works in Artifactory. I will test it there and report back.

Related to #2816 (but in the server side)

**Comment by danimtb** [comment]
Seems this is also affecting Artifactory in the same way

**Comment by danimtb** [comment]
``conan search <reference>`` fails while ``conan install <ref>`` works

**Comment by danimtb** [comment]
Reported in Artifactory Jira: https://www.jfrog.com/jira/browse/RTFACT-19943

**Comment by czoido** [comment]
Closed as this should have been solved by Artifactory 7.12.3
